TerriaJS/terriajs
# Alias `lodash` imports to `lodash-es` using webpack.

### What this PR does

Fixes #5417 

This PR:
- [x] aliases `lodash` imports to `lodash-es` so that we only bundle `lodash-es` even if our dependencies use `lodash`. 
- [x] adds `babel-plugin-lodash` to automatically rewrite `import {x} from "lodash-es"` to the more granular `import x from "lodash-es/x"`.

### Testing

- Open the network tab in 2 browser tabs
- Load both ci.terria.io/next/ & ci.terria.io/alias-lodash-es
- Search for terriamap in the network tab
- Check the size of the bundle called `index.Terriamap.js` - For `/next` this should be around 13.2M and for `/alias-lodash-es` this should be around 12.5M. These are the compressed file size when transferred over the network.


### Checklist
